AcceptanceTests/AcceptanceTests.swift

```diff
@@ -196,89 +258,63 @@ class AcceptanceTests: QuickSpec {
-    }
-    
-    var muterAbortedTestingOutputPath: String { return "\(AcceptanceTests().rootTestDirectory)/muters_aborted_testing_output.txt" }
-    var muterAbortedTestingOutput: String {
-        return contentsOfFileAsString(at: muterAbortedTestingOutputPath)
-    }
+    var muterFilesToMutateOutput: String { contentsOfFileAsString("\(AcceptanceTests().rootTestDirectory)/muters_files_to_mutate_output.txt") }
     
-    var muterHelpOutputPath: String { return "\(AcceptanceTests().rootTestDirectory)/muters_help_output.txt" }
-    var muterHelpOutput: String {
-        return contentsOfFileAsString(at: muterHelpOutputPath)
-    }
+    var muterEmptyStateOutput: String { contentsOfFileAsString("\(AcceptanceTests().rootTestDirectory)/muters_empty_state_output.txt") }
+    var muterAbortedTestingOutput: String { contentsOfFileAsString("\(AcceptanceTests().rootTestDirectory)/muters_aborted_testing_output.txt") }
     
-    var muterLogsRootPath: String { return "\(AcceptanceTests().rootTestDirectory)/muter_logs/" }
+    var muterHelpOutput: String { contentsOfFileAsString("\(AcceptanceTests().rootTestDirectory)/muters_help_output.txt") }
+    var muterInitHelpOutput: String { contentsOfFileAsString("\(AcceptanceTests().rootTestDirectory)/muters_init_help_output.txt") }
+    var muterRunHelpOutput: String { contentsOfFileAsString("\(AcceptanceTests().rootTestDirectory)/muters_run_help_output.txt") }
+
     var muterLogFiles: [String] {
-        return contentsOfDirectory(at: muterLogsRootPath)
+        contentsOfDirectory(muterLogsRootPath)
             .map { muterLogsRootPath + "/" + $0 }
-            .flatMap(contentsOfDirectory(at:))
+            .flatMap(contentsOfDirectory)
     }
     
-    var createdIOSConfigurationPath: String { return "\(AcceptanceTests().rootTestDirectory)/created_iOS_config.json" }
-    var createdIOSConfiguration: Data {
-        return contentsOfFileAsData(at: createdIOSConfigurationPath)
-    }
+    var createdIOSConfiguration: Data { contentsOfFileAsData("\(AcceptanceTests().rootTestDirectory)/created_iOS_config.json") }
+    var createdMacOSConfiguration: Data { contentsOfFileAsData("\(AcceptanceTests().rootTestDirectory)/created_macOS_config.json") }
     
-    var createdMacOSConfigurationPath: String { return "\(AcceptanceTests().rootTestDirectory)/created_macOS_config.json" }
-    var createdMacOSConfiguration: Data {
-        return contentsOfFileAsData(at: createdMacOSConfigurationPath)
-    }
+    var muterLogsRootPath: String { "\(AcceptanceTests().rootTestDirectory)/muter_logs/" }
 }
 
 @available(OSX 10.13, *)
 extension AcceptanceTests {
-    
     func contentsOfLogFile(named fileName: String) -> String {
-        return contentsOfDirectory(at: muterLogsRootPath)
+        return contentsOfDirectory(muterLogsRootPath)
             .first
             .map { muterLogsRootPath + $0 + "/" + fileName }
-            .map (contentsOfFileAsString(at:))!
+            .map (contentsOfFileAsString)!
```

Comment:
```suggestion
            .map(contentsOfFileAsString)!
```
as long as you're touching this line anyway